docs/develop-resources/README.md

```diff
@@ -16,10 +20,15 @@ There are definitely more work need to be done, but we'd like to share the steps
 * [4. Add Controller](./guides/4-add-controller.md)
 * [5. Releases](./guides/5-releases.md)
 
-## Develop by scenraios
+## Develop by scenarios
+
+To determine the best approach for adding support for the resource or field, please check the resource's current status:
+**Check the CRD:** Examine the latest [CRDs](https://github.com/GoogleCloudPlatform/k8s-config-connector/tree/master/crds). The version can be found from the file name. If the resource exists but is in Alpha (and therefore not yet in the [Config Connector API Reference](https://cloud.google.com/config-connector/docs/reference/overview)), follow the [Alpha to Beta promotion guide](./scenarios/alpha-to-beta.md).  This may be the simplest solution.
+**File an issue (if no CRD exists):** If no CRD is found, you'll likely need to create a new resource. Before starting this process, check if an issue is already filed. If not, please file an issue to avoid duplication of effort and allow us to coordinate effectively. If you are willing to take the resource yourself, please assign the issue. 
```

Comment:
I plan to create a issue template for new resource request (to align with the direct reconciliation guide and the PM feature request guide)